tools/generate-templates.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rm -rf test/roles/* || true
+mkdir -p test/roles
+cd test/roles
+
+DRIVER_NAMES=(azure docker gce vagrant podman ec2)
```

Comment:
Is there a way to not hardcode the list ? Each time a new plugin is added, we'll have to edit this script. This can be easily forgotten

Reply:
I can list the folders inside `src/molecule_plugins` but do you think this a good idea ?.

I fear that other folder might be added that are cookiecutter templates

Reply:
I'm not sure what's the best way to do it. If it has to be hardcoded, so be it. Maybe try to do as you said: look at the src/molecule_plugins content ?

Reply:
I will try to find a way to grab the driver names from `pyproject.toml`

```toml
[project.entry-points."molecule.driver"]
azure = "molecule_plugins.azure.driver:Azure"
containers = "molecule_plugins.containers.driver:Container"
docker = "molecule_plugins.docker.driver:Docker"
ec2 = "molecule_plugins.ec2.driver:EC2"
gce = "molecule_plugins.gce.driver:GCE"
podman = "molecule_plugins.podman.driver:Podman"
vagrant = "molecule_plugins.vagrant.driver:Vagrant"
```

Reply:
@apatard I have generated the package names from `pyproject.toml`, I think the code looks ok now. Can you check ?